packages/domain-search/src/components/domain-suggestion/index.tsx

```diff
@@ -39,22 +41,39 @@ export const DomainSuggestion = ( {
 	const { activeQuery } = listContext;
 
 	const domainName = (
-		<span>
+		<span style={ { lineHeight: '24px' } }>
 			<Text
 				size={ activeQuery === 'large' ? 18 : 16 }
-				style={ { marginRight: badges ? '12px' : undefined } }
-				aria-label={ `${ domain }.${ tld }` }
+				style={ {
+					verticalAlign: 'middle',
+					lineHeight: 'inherit',
+					marginRight: badges ? '12px' : undefined,
+				} }
 			>
-				{ domain }
-				<Text size="inherit" weight={ 500 }>
-					.{ tld }
-				</Text>
+				<span
+					aria-label={ `${ domain }.${ tld }` }
+					style={ {
+						wordBreak: 'break-all',
+						// eslint-disable-next-line no-nested-ternary
+						marginRight: notice ? ( activeQuery === 'large' ? '8px' : '4px' ) : undefined,
```

Comment:
Can we avoid inline styles?

Reply:
A bit difficult to handle that with a class. I went for speed, but surely I can revisit it in the future.